features/step_definitions/api/services/api_docs_steps.rb

```diff
@@ -50,6 +50,11 @@
   click_on 'Update spec'
 end
 
+When "they try to update the spec with an invalid Swagger version" do
+  fill_in_api_docs_service_body('{"swagger": "5"}')
```

Comment:
Is it not `"openapi": "5.0.0"` ? That's what the body looks like:
![image](https://user-images.githubusercontent.com/17052241/220359548-4082fcff-1ce2-402e-8e89-ab1bd9f54302.png)